assets/base/i18n/en.json

```diff
@@ -17,6 +17,11 @@
   "channel_header.addMembers": "Add Members",
   "channel_header.directchannel.you": "{displayname} (you) ",
   "channel_header.manageMembers": "Manage Members",
+  "channel_header.notificationPreference": "Mobile notifications",
+  "channel_header.notificationPreference.all": "All",
+  "channel_header.notificationPreference.default": "Default",
+  "channel_header.notificationPreference.mention": "Mentions",
+  "channel_header.notificationPreference.none": "Never",
```

Comment:
are any of this already defined for user setting notifications? maybe we want to re-use those keys instead and only add those that are different? just a thought

Reply:
I tried finding them but couldn't.